docs/community.vmware.vmware_httpapi.rst

```diff
@@ -19,13 +19,49 @@ Synopsis
 - This HttpApi plugin provides methods to connect to VMware vCenter over a HTTP(S)-based APIs.
 
 
-
-
-
-
-
-
-
+Examples:
+---------
+
+Using the 'appliance' API:
+
+- hosts: all
+  connection: httpapi
+  gather_facts: false
+  vars:
+    ansible_network_os: vmware
```

Comment:
There is a problem with the indentation.
ERROR: docs/community.vmware.vmware_httpapi.rst:31:0: Unexpected indentation.

cf: https://9bc0a42696427a0446e6-d7ec1c873a09f376346c7216b9aa819c.ssl.cf5.rackcdn.com/340/2ebe18e2ac91f7eb239cf27c18ebd990195c47b2/check/ansible-test-sanity-vmware/beb7112/job-output.html#l70742